mitiq/observable/observable.py

```diff
@@ -24,6 +24,13 @@
 
 
 class Observable:
+    """A quantum observable typically used to compute its mitigated expectation
+     value.
+
+    It can be defined on a selection of the total qubits in a quantum register.
+    It can be initialized with :class:`.PauliString` objects.
```

Comment:
Do you want to add a proper `__init__` docstring? This content seems more appropriate there.

Reply:
See comment on https://github.com/unitaryfund/mitiq/pull/1050#discussion_r773097781.